lib/pages/chat/chat_input_row.dart

```diff
@@ -92,16 +92,7 @@ class ChatInputRow extends StatelessWidget {
                   margin: const EdgeInsets.only(right: 4.0),
                   icon: Icons.add_circle_outline,
                   onPressed: () async {
-                    final currentPermissionPhotos = await controller.getCurrentPhotoPermission();
-                    final currentPermissionCamera = await controller.getCurrentCameraPermission();
-                    if (currentPermissionPhotos != null && currentPermissionCamera != null) {
```

Comment:
Why need `&&`? How about `||`?

Reply:
raise a bug about it, Huy will fix it later. https://github.com/orgs/linagora/projects/6/views/1?pane=issue&itemId=33533444